tensorflow_datasets/testing/test_data/fake_examples/breast_cancer/breast_cancer.data

```diff
@@ -0,0 +1,13 @@
+1036172,2,1,1,1,2,1,2,1,1,2
```

Comment:
Make sure you have generated this data.

Reply:
I updated the data by generating it manually